tools/champ_blocs/cb_ivr.xml

```diff
@@ -0,0 +1,84 @@
+<tool id="cb_ivr" name="IVR" version="@VERSION@" profile = "20.01">
+    <description>indicator</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="cb_requirements">
+        <requirement type="package" version="2.16">r-rmarkdown</requirement>
+        <requirement type="package" version="0.2.4">r-officedown</requirement>
+        <requirement type="package" version="1.3.2">r-tidyverse</requirement>
+        <requirement type="package" version="0.4.4">r-officer</requirement>
+        <requirement type="package" version="0.8.1">r-flextable</requirement>
+        <requirement type="package" version="3.3.6">r-ggplot2</requirement>
+    </expand>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+            '$__tool_directory__/cb_ivr.r'
+            '$fiche_val'
+            '$input_data'
+            '$__tool_directory__/cb_ivr_site_report_col_scale_loop.Rmd'
+            '$__tool_directory__/loop_col_scale.r'
+            '$output_ivr'
+            '$plots'
+            '$output_report'
+        ]]>
+    </command>
+    <inputs>
+        <expand macro="cb_input"/>
+        <param name="input_data" type="data" format="data" label="Input champbloc_ivr.csv"/>
+    </inputs>
+    <outputs>
+        <data name="output_ivr" from_work_dir="ivr_val_qu_stat.RDS" format="Rdata" label="IVR Rdata"/>
+        <collection type="list" name="plots" label="IVR plot">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.png" visible="false" format="png"/>
+        </collection>
+        <collection type="list" name="output_report" label="Reports">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.docx" directory="results" visible="false" format="docx"/>
```

Comment:
docx? ;) Oo

Reply:
Thanks for the review !! I will do as you suggested for the first 2 comments ! However, i know docx is an ugly output but, the main people who will use those tools want docx files...  '--

Reply:
Understood :-)